i18n-active_record.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.rubyforge_project = '[none]'
 
-  s.add_dependency 'i18n', '>= 0.5.0'
+  s.add_dependency 'i18n', '>= 1.9.0'
```

As per @radar suggestion
```suggestion
  s.add_dependency 'i18n', '>= 1.9.1'
```